tools/rpkg/src/utils.cpp

```diff
@@ -198,6 +198,21 @@ SEXP RApiTypes::ValueToSexp(Value &val) {
 		Rf_setAttrib(res, Rf_install("tzone"), r_ts.Protect(Rf_mkString("UTC")));
 		return res;
 	}
+	case LogicalTypeId::TIME: {
```

Comment:
Is all this R stuff included in this PR on purpose?

Reply:
yes, this is a result of the time-filter pushdown that was not properly done before.